packages/library-legacy/src/programs/vote.ts

```diff
@@ -342,6 +402,23 @@ const VOTE_INSTRUCTION_LAYOUTS = Object.freeze<{
       Layout.voteAuthorizeWithSeedArgs(),
     ]),
   },
+  CompactUpdateVoteState: {
+    index: 12,
+    layout: BufferLayout.struct<VoteInstructionInputData['CompactUpdateVoteState']>([
```

Comment:
You can add this whole layout to `layout.ts` and then just call `Layout.voteCompactUpdateVoteState()` here.